_browser/blockstack_storage.md

```diff
@@ -306,7 +294,7 @@ These are the `UserSession.putFile`, `UserSession.getFile`, and `lookupProfile`
 
             </div>
           </div>
-        </div> : null
+        </div>
       );
     }
```

Comment:
we're implicitly asking to close the component here and remove the `componentWillMount` method. It confused me.

Reply:
Looking at the browser after this step, I get an error:
```
./src/App.js
Attempted import error: './Profile.js' does not contain a default export (imported as 'Profile').
```